CONTRIBUTING.md

```diff
@@ -58,13 +58,7 @@ length of 88; the flake8 pre-commit hook should help with this!
 
 Local links can be written as normal, but external links should be referenced at the
 bottom of the Markdown file for clarity. For example:
-
-```md
-Use a local link to reference the [`README.md`](./README.md) file, but an external link
-for [GOV.UK][gov-uk].
-
-[gov-uk]: https://www.gov.uk/
-```
+Use a local link to reference the [`README.md`](https://github.com/best-practice-and-impact/govcookiecutter/blob/main/README.md) file, but an external link for [GOV.UK](https://www.gov.uk/).
```

Comment:
The way this is written doesn't follow the advice it is giving

Reply:
the links don't work while in '''md ''' I will rearrange them to look like other links but the text has to sit outside '''md '''